hazelcast-client/src/test/java/com/hazelcast/client/internal/properties/HazelcastPropertiesTest.java

```diff
@@ -1,7 +1,8 @@
-package com.hazelcast.client.config;
+package com.hazelcast.client.internal.properties;
 
```

Comment:
Good question, but a good catch, the test seems still to be in the wrong package.